pydantic/color.py

```diff
@@ -20,21 +20,28 @@
 if TYPE_CHECKING:  # pragma: no cover
     from .types import CallableGenerator
 
-
 ColorTuple = Union[Tuple[int, int, int], Tuple[int, int, int, float]]
 ColorType = Union[ColorTuple, str]
 HslColorTuple = Union[Tuple[float, float, float], Tuple[float, float, float, float]]
 
 
-class RGBA(NamedTuple):
+class RGBA:
```

Comment:
I spent a while trying to come up with a different implementation of this that was more directly subclassing Tuple (so I wouldn't have to rework all the callsites), but couldn't get it to play nice with Cython. This works with no changes, but does duplicate the data.

The duplication could be undone by having r, g, b, and alpha be properties obtained by indexing into the tuple, but I'm not sure if its worth the extra calls (but maybe Cython would be able to optimize them away anyway). Or maybe there is an unrelated implementation that is just universally better.

Let me know if you want me to change anything.